test/BnomialToken1.test.js

```diff
@@ -0,0 +1,56 @@
+const { BigNumber } = require("@ethersproject/bignumber");
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+
+
+describe("BinomialToken1", () => {

```

Comment:
The test coverage for the token contract is just 50%. We should always strive to reach 100% on our smart contracts. To see the missing coverage run:
```
npm run coverage
```

After that, you can open the `coverage/index.html` file to see which lines/branches are not covered.

Reply:
any clues on how to test _pause, _unpause, and _burn functions in test coverage?

Reply:
Yes, I've implemented some of these for the Underfitted NFT project: https://github.com/underfitted/underfitted-membership-nft/blob/678fdfd4eb5606b886ce4b0bb70f9e7361243c02/test/common.js#L84

Regarding `burn` - I think you can just check that the balance of a wallet decreased by the amount of burned tokens.

Reply:
@haltakov  all changes completed.